core/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,6 +43,7 @@
     <property name="adjustExternalResourceFields" value="${lineage.adjust.external.resource.fields}"/>
     <property name="generateSubGraphs" value="${lineage.generate.subgraphs}"/>
     <property name="consolidateSubGraphs" value="${lineage.consolidate.subgraphs}"/>
+    <property name="externalResourceCacheExpireTime" value="${lineage.external.resource.cache.expire.time}"/>
```

Comment:
update to pentaho.metaverse.cfg is in PR https://github.com/pentaho/pentaho-karaf-assembly/pull/642